backend/docker/demo/Dockerfile

```diff
@@ -16,13 +14,21 @@ RUN set -x \
     && cp /build/coreroller/backend/bin/rollerd /coreroller \
     && cd /build/coreroller/frontend \
     && npm install \
-    && npm run build \
-    && cp -R built/* /coreroller/static \
-    && apk del .build-deps \
-    && rm -rf /var/cache/apk /go /build /root/.npm /tmp/npm*
+    && npm run build 
 
-ADD docker-entrypoint.sh /
+FROM alpine:3.7
+
+RUN apk --no-cache add ca-certificates tzdata curl \
+    && apk --no-cache add postgresql postgresql-contrib \
+    && mkdir -p /coreroller/static \
+    && mkdir -p /run/postgresql \
+    && chown -R postgres:postgres /run/postgresql \
+    && curl -o /usr/local/bin/gosu -sSL "https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64" \
+    && chmod +x /usr/local/bin/gosu
 
+COPY --from=builder /build/coreroller/backend/bin/rollerd /coreroller/
+COPY --from=builder /build/coreroller/frontend/built/* /coreroller/static/
```

Comment:
Pretty sure this line causes an issue and flattens everything that is copied:

```
/coreroller/static # ls
FontAwesome.otf                        lato-light-webfont.woff2
animated_dots.gif                      lato-regular-webfont.eot
bootstrap.css                          lato-regular-webfont.svg
btn_loading.gif                        lato-regular-webfont.ttf
coreos.png                             lato-regular-webfont.woff
docker.png                             lato-regular-webfont.woff2
empty_bg.png                           loaders.min.css
favicon.ico                            loadingCR.gif
font-awesome.css                       main.js
fontawesome-webfont.eot                mini_loading.gif
fontawesome-webfont.svg                other.png
fontawesome-webfont.ttf                robotocondensed-light-webfont.eot
fontawesome-webfont.woff               robotocondensed-light-webfont.svg
fontawesome-webfont.woff2              robotocondensed-light-webfont.ttf
glyphicons-halflings-regular.eot       robotocondensed-light-webfont.woff
glyphicons-halflings-regular.svg       robotocondensed-light-webfont.woff2
glyphicons-halflings-regular.ttf       robotocondensed-regular-webfont.eot
glyphicons-halflings-regular.woff      robotocondensed-regular-webfont.svg
glyphicons-halflings-regular.woff2     robotocondensed-regular-webfont.ttf
index.html                             robotocondensed-regular-webfont.woff
lato-bold-webfont.eot                  robotocondensed-regular-webfont.woff2
lato-bold-webfont.svg                  rocket.png
lato-bold-webfont.ttf                  styles.css
lato-bold-webfont.woff                 user1.png
lato-bold-webfont.woff2                user2.png
lato-light-webfont.eot                 user3.png
lato-light-webfont.svg                 user4.png
lato-light-webfont.ttf                 user5.png
lato-light-webfont.woff                user6.png

```
>If is a directory, the entire contents of the directory are copied, including filesystem metadata. Note: The directory itself is not copied, just its contents.

Ref: https://docs.docker.com/engine/reference/builder/#/copy

Will file a PR for fixing this

Reply:
This results in a 404 on all static resources on the container.

Reply:
Thanks for looking into this @captn3m0, a PR would be welcome :)